### PR TITLE
RESTWS-753: Support OpenMRS platform 2.3.x

### PR DESCRIPTION
**Pull request by ODORA0**
issue worked on: https://issues.openmrs.org/browse/RESTWS-753

### CODE REVIEW

File: omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/PersonResource1_10.java [hunk 1]
```diff
@@ -24,58 +24,58 @@
...
         "1.10.4 - 1.10.6" })
 public class PersonResource1_10 extends PersonResource1_8 {
-
-    /**
-     * @see DelegatingCrudResource#getRepresentationDescription(Representation)
-     */
-    @Override
-    public DelegatingResourceDescription getRepresentationDescription(Representation rep) {
-        DelegatingResourceDescription description = super.getRepresentationDescription(rep);
-        if (description != null) {
```
Comment (dkayiwa) [inline omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/PersonResource1_10.java]: Do we need these changes?

File: omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/DrugController1_10Test.java [hunk 2]
```diff
@@ -97,19 +97,19 @@ public void getDrugsByMapping_shouldReturnADrugThatMatchesTheCodeAndSourceAndThe
 		SimpleObject results = deserialize(handle(newGetRequest(getURI(), new Parameter("s", "getDrugsByMapping"),
 		    new Parameter("code", "CD41003"), new Parameter("source", sourceUuid), new Parameter("preferredMapTypes",
 		            mapTypeUuids))));
-
+
```
Comment (dkayiwa) [inline omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/DrugController1_10Test.java]: Do we need these formatting changes?

File: omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/PersonResource1_10Test.java [hunk 3]
```diff
@@ -15,31 +15,31 @@
...
-    }
-
-    @Override
-    public void validateDefaultRepresentation() throws Exception {
-        super.validateDefaultRepresentation();
-        assertPropEquals("birthtime", getObject().getBirthtime());
-    }
-
-    @Override
-    public void validateFullRepresentation() throws Exception {
```
Comment (dkayiwa) [inline omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/PersonResource1_10Test.java]: Do we need these changes?

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java [hunk 4]
```diff
@@ -50,7 +50,7 @@
  * {@link Resource} for Person, supporting standard CRUD operations
  */
 @Resource(name = RestConstants.VERSION_1 + "/person", order = 2, supportedClass = Person.class, supportedOpenmrsVersions = {
-	"1.8.*", "1.9.*", "1.10.0 - 1.10.3" })
+        "1.8.*", "1.9.*", "1.10.0 - 1.10.3" })
```
Comment (dkayiwa) [inline omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java]: Do we need these changes?

File: omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PersonAddressController1_8Test.java [hunk 5]
```diff
@@ -38,7 +38,7 @@
 public class PersonAddressController1_8Test extends MainResourceControllerTest {
 	
 	private static final String ADDRESS_TEST_UUID = "address-test-uuid";
-
+
```
Comment (dkayiwa) [inline omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PersonAddressController1_8Test.java]: Do we need these changes?

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java (hunk 4 above)
Comment (ODORA0) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java]: @dkayiwa none of those were intended, surprised to find them too

File: omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java [hunk 6]
```diff
@@ -1,555 +0,0 @@
-/**
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
- * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
- * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
```
Comment (dkayiwa) [inline omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java]: Why did you delete this file and others?

Comment (ODORA0) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java]: For the most part they weren't part of the ticket requirements and thus making changes for them would have not been needed so I thought deleting would be an option.

Comment (ODORA0) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java]: For the most part they weren't part of the ticket requirements and thus making changes for them would have not been needed so I thought deleting would be an option.

Comment (dkayiwa) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java]: Deleting a file means deleting code. Was that your intention?

Comment (ODORA0) [reply omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonResource1_8.java]: Oops!! No not at all my intention, I think that's the reason for the travis failure

**Comment by dkayiwa** [comment]
Closing because of lack of activity on this pr
